code/modules/security_levels/security_levels.dm

```diff
@@ -67,7 +68,7 @@
 						FA.add_overlay(image('icons/obj/monitors.dmi', "overlay_delta"))
 				if(!delta_timer_id)
 					delta_alarm()
-		SSsmartlight.check_nightshift() // Night shift mode turns off if security level is raised to red or above
+				SSsmartlight.update_mode(light_modes_by_name["Code Delta"], TRUE)
```

Comment:
ломает старое поведение для красного кода. Но не то, чтоб у нас сейчас активно использовался ночной свет, пусть пока хоть комментом останется.

```suggestion
				SSsmartlight.update_mode(light_modes_by_name["Code Delta"], TRUE)
			// commented in favor of deltacode above, also because we don't use NS actively atm. Need to revisit this
			//SSsmartlight.check_nightshift() // Night shift mode turns off if security level is raised to red or above
```

идеально было бы обновить ``check_nightshift`` и сделать сабсистеме отдельный прок, переключающий свет в зависимости от тревоги, но не буду грузить